### ISSUE TITLE
missing GensimFeaturizer module after pip install git+ 

### ISSUE THREAD

**Issue by yari0757** [RasaHQ/rasa-nlu-examples#32]
I put Gensimfeaturizer in rasa nlu pipeline and got this message:

**Exception: Failed to find class 'GensimFeaturizer' in module 'rasa_nlu_examples.featurizers.dense'.**

After try uninstall and install with pip git and python pip git, clone master branch with last commit and ... but the problem not solved.
finally, i found that the class .py file not present so i  copy/paste gensim_featurizer.py in package installed folder(/rasa_nlu_example/featurizer/dense) and it did work!

**Comment by koaning** [comment]
d0h! 

Will make a fix. Thanks for reporting!

**Comment by koaning** [comment]
Fixed. Thanks for reporting! https://github.com/RasaHQ/rasa-nlu-examples/pull/33

**Comment by koaning** [comment]
Could you rephrase what you mean with `class.py` file?

**Comment by yari0757** [comment]
> Could you rephrase what you mean with `class.py` file?

GensimFeaturizer class in gensim_featurizer.py file!

**Comment by koaning** [comment]
Gotya. I think that *should* be fixed now. Could you try again? Should it fail, could you copy the full traceback or the error as well as the command that you're running?

**Comment by koaning** [comment]
Closing this due to inactivity.
